### PR TITLE
Add icons when not signed in to Azure

### PR DESCRIPTION
**Pull request by EricJizbaMSFT**
I was adding "Create New Project" per https://github.com/microsoft/vscode-azurefunctions/issues/1322 and thought some icons might be nice:

Screen Shot 2019-07-19 at 3 43 12 PM
Screen Shot 2019-07-19 at 3 43 28 PM

Also just cleaned up the icon code.

### CODE REVIEW

File: ui/src/treeDataProvider/AzureAccountTreeItemBase.ts [hunk 1]
```diff
@@ -148,6 +149,14 @@ export abstract class AzureAccountTreeItemBase extends AzExtParentTreeItem imple
         return undefined;
     }
 
+    public compareChildrenImpl(item1: AzExtTreeItem, item2: AzExtTreeItem): number {
```
Comment (StephenWeatherford) [inline ui/src/treeDataProvider/AzureAccountTreeItemBase.ts]: Is this related?

Comment (EricJizbaMSFT) [reply ui/src/treeDataProvider/AzureAccountTreeItemBase.ts]: Yeah without it the items in the above screenshot would be sorted alphabetically:

- Create a Free Azure Account...
- Create New Local Project...
- Sign in to Azure...

I don't have a strong preference on the order _other than_ the two Azure ones should be next to each other. Which unfortunately is exactly what happens with alphabetical sorting.
